ClosedXML/Excel/XLWorkbook.cs

```diff
@@ -664,6 +664,8 @@ public XLWorkbook()
         internal XLWorkbook(String file, Boolean asTemplate)
             : this(new LoadOptions())
         {
+            if (!asTemplate) throw new NotSupportedException();
```

Comment:
Why don't we fallback to ordinary loading? As a user, I would expect these two calls producing identical result:
```
var wb = new XLWorkbook(fileName);
var wb = new XLWorkbook(fileName, asTemplate: false);
```